RecoParticleFlow/PFProducer/src/PFAlgo.cc

```diff
@@ -23,7 +23,9 @@ PFAlgo::PFAlgo(double nSigmaECAL,
       calibration_(calibration),
       thepfEnergyCalibrationHF_(thepfEnergyCalibrationHF),
       connector_() {
-  pfmu_ = std::make_unique<PFMuonAlgo>(pset);
+  const edm::ParameterSet pfMuonAlgoParams = pset.getParameter<edm::ParameterSet>("PFMuonAlgoParameters");
+  bool postMuonCleaning = pset.existsAs<bool>("postMuonCleaning") ? pset.getParameter<bool>("postMuonCleaning") : false;
```

Comment:
isn't ```PFAlgo``` used only by the ```PFProducer```?
If so, the ```postMuonCleaning``` was already added via the fillDescriptions there

Reply:
Hmm. PFAlgo's fillDescriptions just has this parameter then? I am not sure I like the idea, but if it's really preferred, I think I can do it.

Reply:
I thought this was the case from 
https://github.com/cms-sw/cmssw/pull/28110/files#diff-24b7d20f1967c466ee0593961eb6c237R325

Reply:
now i see what you mean. thank you.